omnia.sh

```diff
@@ -620,7 +620,12 @@ validate_oim() {
         exit 1
     fi
 
-
+    oim_timezone=$(timedatectl | grep "Time zone" | awk '{print $3}')
```

Comment:
grep time zone is not safe as it can be changed syntax later. can we have any other way to get timezone of oim in any file or any proper command. case sensitive to be handled?